CMakeLists.txt

```diff
@@ -178,6 +178,9 @@ set(CMAKE_VERBOSE_MAKEFILE ON)
 include_directories(${PROJECT_SOURCE_DIR}/src)
 include_directories(${PROJECT_BINARY_DIR}/src)
 
+include_directories(${PROJECT_SOURCE_DIR}/thirdparty/installed/include/hdfs)
```

Comment:
This line should fail in the docker environment where dependencies are installed in /usr/local